### DISCUSSION TITLE
is there a reason why str() and to_string give diffenret results on a CRS ?

### DISCUSSION THREAD

**Discussion by 12rambau** [pyproj4/pyproj#1118] [Q&A] [answered]
I tried the following: 

```python
from pyproj import CRS 

CRS(4326).to_string()
>>>> 'EPSG:4326' 

str(CRS(4326))
>>>> 'epsg:4326'
```` 

Any reason why these are not the same ?

**Comment by snowman2** [comment, marked as answer]
[to_string](https://pyproj4.github.io/pyproj/stable/api/crs/crs.html#pyproj.crs.CRS.to_string) attempts to convert to an authority string and falls back to the user input. It has more overhead to run. Calling `str` calls the `__str__` method which returns the user input directly.
